drivers/telescope/lx200classic.cpp

```diff
@@ -412,7 +396,7 @@ bool LX200Classic::Park()
     IDSetSwitch(&UnparkAlignmentSP, nullptr);
     saveConfig(true, UnparkAlignmentSP.name);
 
-    if (!Goto(equatorialPos.ra / 15.0, equatorialPos.dec))
+    if (!Goto(parkRA, parkDEC))
```

Comment:
This is all so much cleaner with the utility function handling the mathy  bits

Reply:
I agree. Much nicer.